macos/FluentUI/Button/Button.swift

```diff
@@ -228,16 +233,15 @@ open class Button: NSButton {
 		}
 	}
 
-	private var restBackgroundColor: NSColor {
-		return fillColor
-	}
-
+	@objc public var restBackgroundColor: NSColor? = nil
+	@objc public var cornerRadius: CGFloat = 3
```

Comment:
Why do we now need to expose the cornerRadius? I didn't know modifying it was part of the toolkit

Reply:
You're right, my mistake...